scripts/publish-deb

```diff
@@ -121,6 +121,47 @@ update_repo() {
     popd > /dev/null
 }
 
+reduce_dir_size() {
```

Comment:
Note for the reviewer: you might notice that this function is duplicated in two files. I actually wanted to add it to a different file that could be sourced relatively to this script but it appears that this cannot be done reliably in Bash because it depends on where the script is located and would make it unportable. Rather than figuring out Bash I thought about just copy pasting it